System/StandardIncludes.h

```diff
@@ -5,50 +5,57 @@
 #define _HAS_AUTO_PTR_ETC 1
 
 // Inclusion of relevant C++ Standard Library headers.
-#include <cstdlib>
```

Comment:
I forgot to ask this before but why did you rearrange these?

Reply:
Don't entirely remember, have reverted that now. I also removed the buffer safe __STLIBC_WANTS_EXT1__ stuff since that only does anything in C and the according headers (<string.h>), and for c++ is purely a vsc++ (MSVC) feature, so doesn't exist anywhere else anyways. Also since they're used unnecessarily almost everywhere (except for the uses of memcpy_s in the networking code), it's not a big deal.